src/Symfony/Bundle/FrameworkBundle/Resources/config/services.xml

```diff
@@ -17,10 +18,13 @@
             <argument type="service" id="controller_resolver" />
             <argument type="service" id="request_stack" />
             <argument type="service" id="argument_resolver" />
+            <tag name="container.hot_path" />
         </service>
         <service id="Symfony\Component\HttpKernel\HttpKernelInterface" alias="http_kernel" />
 
-        <service id="request_stack" class="Symfony\Component\HttpFoundation\RequestStack" public="true" />
+        <service id="request_stack" class="Symfony\Component\HttpFoundation\RequestStack" public="true">
+            <tag name="container.hot_path" />
```

Comment:
do we actually need this ? HttpKernel propagates it anyway (and if HttpKernel does not use the stack anymore, it is not hot anymore)